events/importer/yso.py

```diff
@@ -248,6 +248,9 @@ def save_alt_label(self, syncher, graph, label):
         if label.language is None:
             logger.error('Error: {} has no language'.format(label))
             return None
+        if label.language not in YsoImporter.supported_languages:
```

Comment:
Should this be: `logger.error('Error: {} has not supported language: {}'.format(label, label.language))`

Note. I did not test if that my code works.